docs/modules_packages_crates/crates_and_packages.md

```diff
@@ -14,8 +13,12 @@ Crates can contain modules, and the modules may be defined in other files that g
 
 A Noir crate can come in one of two forms: a binary crate or a library crate.
 
+#### Binaries
+
 _Binary crates_ are programs which you can compile to an ACIR circuit which you can then create proofs against. Each must have a function called `main` that defines the ACIR circuit which is to be proved.
 
+#### Libraries
```

Comment:
Do we have contract crates or is that not relevant for this page?

Reply:
Yes, good call. Can you write a sentence or two about how they differ from libraries and binaries? Or direct me to the best person to?